forch/simple_auth_state_machine.py

```diff
@@ -101,7 +109,15 @@ def received_radius_reject(self):
             self._auth_callback(self.src_mac, self.UNAUTH, None, None)
 
     def handle_sm_timer(self):
-        """Handle timer timeout and check.trigger timeout behavior of states"""
+        """
+        Handle timer timeout and check.trigger timeout behavior of states:
+        * If the device is in request state and the RADIUS request times out, device will be
```

Comment:
consolidate each to one line for readability -- more is not always better. E.g.:
* REQUEST & timeout --> UNAUTH
* UNAUTH --> no change